app/scripts/templates/settings/recovery_codes.mustache

```diff
@@ -36,15 +36,15 @@
 
             {{^isIos}}
                 <div class="button-row recovery-code-options">
-                    <a class="recovery-code-option download-codes">
+                    <a href="#" class="recovery-code-option download-codes">
```

Comment:
can these be "<button>" instead?

Reply:
Tried using `href=''"` and the page reloads instead of downloading the codes :/

Reply:
sorry, github ate the 'button' tag

Reply:
Made https://github.com/mozilla/fxa-content-server/issues/6257 to track this.